gradle/dependencies.gradle

```diff
@@ -50,21 +54,21 @@ ext {
             mockk                  : "io.mockk:mockk:${versions.mockk}",
             robolectric            : "org.robolectric:robolectric:${versions.robolectric}",
 
-            testRunner             : "com.android.support.test:runner:${versions.testRunner}",
-            testRules              : "com.android.support.test:rules:${versions.testRunner}",
-            testEspressoCore       : "com.android.support.test.espresso:espresso-core:${versions.espresso}",
-            testEspressoIntents    : "com.android.support.test.espresso:espresso-intents:${versions.espresso}",
-            testEspressoContrib    : "com.android.support.test.espresso:espresso-contrib:${versions.espresso}",
-            testUiAutomator        : "com.android.support.test.uiautomator:uiautomator-v18:${versions.uiAutomator}",
+            testRunner             : "androidx.test.ext:junit:${versions.testRunner}",
+            testRules              : "androidx.test:rules:${versions.testRunner}",
+            testEspressoCore       : "androidx.test.espresso:espresso-core:${versions.espresso}",
+            testEspressoIntents    : "androidx.test.espresso:espresso-intents:${versions.espresso}",
+            testEspressoContrib    : "androidx.test.espresso:espresso-contrib:${versions.espresso}",
+            testUiAutomator        : "androidx.test.uiautomator:uiautomator:${versions.uiAutomator}",
             appCenter              : "com.microsoft.appcenter:espresso-test-extension:${versions.appcenter}",
             commonsIO              : "commons-io:commons-io:${versions.commonsIO}",
-            supportAnnotations     : "com.android.support:support-annotations:${versions.supportLib}",
-            supportAppcompatV7     : "com.android.support:appcompat-v7:${versions.supportLib}",
-            supportFragmentV4      : "com.android.support:support-fragment:${versions.supportLib}",
+            supportAnnotations     : "androidx.annotation:annotation:${versions.annotation}",
+            supportAppcompatV7     : "androidx.appcompat:appcompat:${versions.appCompat}",
+            supportFragmentV4      : "androidx.fragment:fragment:${versions.fragment}",
             supportUtilV4          : "com.android.support:support-core-utils:${versions.supportLib}",
```

Comment:
Do we still need this dependency? if yes maybe also update this dependency to androidx.

Reply:
nice catch!